.github/workflows/test.yml

```diff
@@ -21,8 +21,6 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby_version }}
           bundler-cache: true
-      - name: Install dependencies
-        run: bundle install --jobs=3 --retry=3
```

Comment:
`bundler-cache: true` on the previous step already runs a `bundle install` and caches the gems, so this isn't necessary.